src/box/errcode.h

```diff
@@ -141,287 +141,313 @@ struct errcode_record {
+	_(ER_UPDATE_FIELD, 29,			"Field %s UPDATE error: %s", "field", STRING, "details", STRING) \
 	_(ER_FUNCTION_TX_ACTIVE, 30,		"Transaction is active at return from function") \
-	_(ER_KEY_PART_COUNT, 31,		"Invalid key part count (expected [0..%u], got %u)") \
-	_(ER_PROC_LUA, 32,			"%s") \
-	_(ER_NO_SUCH_PROC, 33,			"Procedure '%.*s' is not defined") \
-	_(ER_NO_SUCH_TRIGGER, 34,		"Trigger '%s' doesn't exist") \
-	_(ER_NO_SUCH_INDEX_ID, 35,		"No index #%u is defined in space '%s'") \
-	_(ER_NO_SUCH_SPACE, 36,			"Space '%s' does not exist") \
-	_(ER_NO_SUCH_FIELD_NO, 37,		"Field %d was not found in the tuple") \
-	_(ER_EXACT_FIELD_COUNT, 38,		"Tuple field count %u does not match space field count %u") \
-	_(ER_FIELD_MISSING, 39,			"Tuple field %s required by space format is missing") \
+	_(ER_KEY_PART_COUNT, 31,		"Invalid key part count (expected [0..%u], got %u)", "max", UINT, "value", UINT) \
+	_(ER_PROC_LUA, 32,			"%s", "details", STRING) \
+	_(ER_NO_SUCH_PROC, 33,			"Procedure '%s' is not defined", "func", STRING) \
+	_(ER_NO_SUCH_TRIGGER, 34,		"Trigger '%s' doesn't exist", "trigger", STRING) \
+	_(ER_NO_SUCH_INDEX_ID, 35,		"No index #%u is defined in space '%s'", "index_id", UINT, "space", STRING) \
+	_(ER_NO_SUCH_SPACE, 36,			"Space '%s' does not exist", "space", STRING) \
+	_(ER_NO_SUCH_FIELD_NO, 37,		"Field %d was not found in the tuple", "fieldno", INT) \
+	_(ER_EXACT_FIELD_COUNT, 38,		"Tuple field count %u does not match space field count %u", "actual", UINT, "expected", UINT) \
+	_(ER_FIELD_MISSING, 39,			"Tuple field %s required by space format is missing", "field", STRING) \
 	_(ER_WAL_IO, 40,			"Failed to write to disk") \
 	_(ER_MORE_THAN_ONE_TUPLE, 41,		"Get() doesn't support partial keys and non-unique indexes") \
 	_(ER_ACCESS_DENIED, 42,			"%s access to %s '%s' is denied for user '%s'") \
-	_(ER_CREATE_USER, 43,			"Failed to create user '%s': %s") \
-	_(ER_DROP_USER, 44,			"Failed to drop user or role '%s': %s") \
-	_(ER_NO_SUCH_USER, 45,			"User '%s' is not found") \
-	_(ER_USER_EXISTS, 46,			"User '%s' already exists") \
+	_(ER_CREATE_USER, 43,			"Failed to create user '%s': %s", "user", STRING, "details", STRING) \
+	_(ER_DROP_USER, 44,			"Failed to drop user or role '%s': %s", "user", STRING, "details", STRING) \
+	_(ER_NO_SUCH_USER, 45,			"User '%s' is not found", "user", STRING) \
+	_(ER_USER_EXISTS, 46,			"User '%s' already exists", "user", STRING) \
 	_(ER_CREDS_MISMATCH, 47,		"User not found or supplied credentials are invalid") \
-	_(ER_UNKNOWN_REQUEST_TYPE, 48,		"Unknown request type %u") \
-	_(ER_UNKNOWN_SCHEMA_OBJECT, 49,		"Unknown object type '%s'") \
-	_(ER_CREATE_FUNCTION, 50,		"Failed to create function '%s': %s") \
-	_(ER_NO_SUCH_FUNCTION, 51,		"Function '%s' does not exist") \
-	_(ER_FUNCTION_EXISTS, 52,		"Function '%s' already exists") \
+	_(ER_UNKNOWN_REQUEST_TYPE, 48,		"Unknown request type %u", "request_type", UINT) \
+	_(ER_UNKNOWN_SCHEMA_OBJECT, 49,		"Unknown object type '%s'", "object_type", STRING) \
```

Comment:
I'd stick with "value" for "unknown something" type of errors.

Reply:
Fixed.

<details>
<summary>diff</summary>

```diff
diff --git a/src/box/errcode.h b/src/box/errcode.h
index 0da2d95a2..6e7658c05 100644
--- a/src/box/errcode.h
+++ b/src/box/errcode.h
@@ -192,8 +192,8 @@ struct errcode_record {
 	_(ER_NO_SUCH_USER, 45,			"User '%s' is not found", "user", STRING) \
 	_(ER_USER_EXISTS, 46,			"User '%s' already exists", "user", STRING) \
 	_(ER_CREDS_MISMATCH, 47,		"User not found or supplied credentials are invalid") \
-	_(ER_UNKNOWN_REQUEST_TYPE, 48,		"Unknown request type %u", "request_type", UINT) \
-	_(ER_UNKNOWN_SCHEMA_OBJECT, 49,		"Unknown object type '%s'", "object_type", STRING) \
+	_(ER_UNKNOWN_REQUEST_TYPE, 48,		"Unknown request type %u", "value", UINT) \
+	_(ER_UNKNOWN_SCHEMA_OBJECT, 49,		"Unknown object type '%s'", "value", STRING) \
 	_(ER_CREATE_FUNCTION, 50,		"Failed to create function '%s': %s", "func", STRING, "details", STRING) \
 	_(ER_NO_SUCH_FUNCTION, 51,		"Function '%s' does not exist", "func", STRING) \
 	_(ER_FUNCTION_EXISTS, 52,		"Function '%s' already exists", "func", STRING) \
@@ -337,7 +337,7 @@ struct errcode_record {
 	_(ER_SQL_KEYWORD_IS_RESERVED, 183,	"At line %d at or near position %d: keyword '%s' is reserved. Please use double quotes if '%s' is an identifier.", \
 						"line", INT, "column", INT, "keyword", STRING, "", STRING) \
 	_(ER_SQL_SYNTAX_NEAR_TOKEN, 184,	"Syntax error at line %d near '%s'", "line", INT, "token", STRING) \
-	_(ER_SQL_UNKNOWN_TOKEN, 185,		"At line %d at or near position %d: unrecognized token '%s'", "line", INT, "column", INT, "token", STRING) \
+	_(ER_SQL_UNKNOWN_TOKEN, 185,		"At line %d at or near position %d: unrecognized token '%s'", "line", INT, "column", INT, "value", STRING) \
 	_(ER_SQL_PARSER_GENERIC, 186,		"%s", "details", STRING) \
 	/* ER_SQL_ANALYZE_ARGUMENT, 187, Unused */ \
 	_(ER_SQL_COLUMN_COUNT_MAX, 188,		"Failed to create space '%s': space column count %d exceeds the limit (%d)", "space", STRING, "value", INT, "max", INT) \
@@ -405,7 +405,7 @@ struct errcode_record {
 	_(ER_ITERATOR_POSITION, 247,		"Iterator position is invalid") \
 	_(ER_DEFAULT_VALUE_TYPE, 248,		"Type of the default value does not match tuple field %s type: expected %s, got %s", \
 						"field", STRING, "expected", STRING, "actual", STRING) \
-	_(ER_UNKNOWN_AUTH_METHOD, 249,		"Unknown authentication method '%s'", "auth_method", STRING) \
+	_(ER_UNKNOWN_AUTH_METHOD, 249,		"Unknown authentication method '%s'", "value", STRING) \
 	_(ER_INVALID_AUTH_DATA, 250,		"Invalid '%s' data: %s", "auth_method", STRING, "details", STRING) \
 	_(ER_INVALID_AUTH_REQUEST, 251,		"Invalid '%s' request: %s", "auth_method", STRING, "details", STRING) \
 	_(ER_WEAK_PASSWORD, 252,		"Password doesn't meet security requirements: %s", "details", STRING) \
@@ -416,7 +416,7 @@ struct errcode_record {
 	_(ER_AUTH_DELAY, 257,			"Too many authentication attempts") \
 	_(ER_AUTH_REQUIRED, 258,		"Authentication required") \
 	_(ER_SQL_SEQ_SCAN, 259,			"Scanning is not allowed for %s", "object", STRING) \
-	_(ER_NO_SUCH_EVENT, 260,		"Unknown event %s", "even", STRING) \
+	_(ER_NO_SUCH_EVENT, 260,		"Unknown event %s", "value", STRING) \
 	_(ER_BOOTSTRAP_NOT_UNANIMOUS, 261,	"Replica %s chose a different bootstrap leader %s", "replica", STRING, "bootstrap_leader", STRING) \
 	_(ER_CANT_CHECK_BOOTSTRAP_LEADER, 262,	"Can't check who replica %s chose its bootstrap leader", "replica", STRING) \
 	_(ER_BOOTSTRAP_CONNECTION_NOT_TO_ALL, 263, "Some replica set members were not specified in box.cfg.replication") \

```

</details>